src/http_rest_api_handler.cpp

```diff
@@ -285,60 +283,62 @@ Status parseBinaryInput(::inference::ModelInferRequest_InferInputTensor* input,
     return StatusCode::OK;
 }
 
+static std::string contentFieldNotEmptyErrorMessage = " contents is not empty. Content field should be empty when using binary inputs extension.";
+
 Status validateContentFieldsEmptiness(::inference::ModelInferRequest_InferInputTensor* input) {
     if (input->datatype() == "FP32") {
         if (input->contents().fp32_contents_size() > 0) {
-            SPDLOG_ERROR("FP32 contents is not empty. Content field should be empty when using binary inputs extension.");
+            SPDLOG_DEBUG("FP32" + contentFieldNotEmptyErrorMessage);
```

Comment:
Whats's happening under the hood in SPDLOG?
I asssume there is some kind of string concatenation and right now we switched from 2 to 3 concatenations. While code looks cleaner maybe we should have macro here instead of std::string so we still gain benefits of once initialized cstr's?

Reply:
How about:
```
SPDLOG_DEBUG("FP32 {}", contentFieldNotEmptyErrorMessage);
```

Reply:
This is not critical path if I'm not mistaken